src/main/java/kostka/moviecatalog/dto/RuntimeConfigDto.java

```diff
@@ -0,0 +1,23 @@
+package kostka.moviecatalog.dto;
+
+import org.hibernate.validator.constraints.Range;
+
+import javax.validation.constraints.NotNull;
+
+public final class RuntimeConfigDto {
+    @NotNull
+    @Range(min = 1, max = 10)
+    private Integer visibleMoviesValue;
```

Comment:
Idea (because I don't know if this can work here :D ): if it is NotNull, I'd suggest to use primitive integer type.